docs/concepts/Auditing-Packages.md

```diff
@@ -0,0 +1,108 @@
+
+> [!IMPORTANT]
+> Security auditing at restore time is available in .NET 8 Preview 4+ and Visual Studio 17.7 Preview 2+.
+
+## Running a security audit with `restore`
+
+The `restore` command automatically runs when you do a common package operation such as loading a project for the first time, adding a new package, updating a package version, or removing a package from your project in your favorite IDE. A description of your dependencies is checked against a report of known vulnerabilities on the [GitHub Advisory Database](https://github.com/advisories?query=type%3Areviewed+ecosystem%3Anuget).
+
+> [!IMPORTANT]
+> For Audit to check packages, a package source that provides a vulnerability database must be used.
+> NuGet.org's V3 URL is one such example (https://api.nuget.org/v3/index.json), but note that NuGet.org's V2 endpoint does not.
+
+> [!NOTE]
+> .NET 8 preview 5+ enables Audit by default, but Visual Studio 17.7 does not ship .NET 8.
+> To opt-in to Audit explicitly, set `<NuGetAudit>true</NuGetAudit>` in your project file, or a *Directory.Build.props* file.
+
+1. On the command line, navigate to your project or solution directory.
+2. Ensure your project or solution contains a `.csproj` or `packages.config` files.
+3. Type `dotnet restore`.
+4. Review the audit report and address the known security vulnerabilities.
+
+## Reviewing and acting on the security audit report
+
+Running `dotnet restore` will produce a report of security vulnerabilities with the affected package name, the severity of the vulnerability, and a link to the advisory for more details.
+
+### Security vulnerabilities found with updates
+
+If security vulnerabilities are found and updates are available for the package, you can either:
+
+- Edit the `.csproj` or other package version location(`Directory.Packages.props`) with a newer version containing a security fix.
+- Use the NuGet package manager user interface in Visual Studio to update the individual package.
+- Run the `dotnet add package` command with the respective package ID to update to the latest version.
+
+### Security vulnerabilities found with no updates
+
+In the case that a known vulnerability exists in a package without a security fix, you can do the following.
+
+- Check for any mitigating factors outlined in the advisory report.
+- Use a suggested package if the package is marked deprecated or is abandoned.
+- If the package is open source, consider contributing a fix.
+- Open an issue in the package's issue tracker.
+
+#### Check for mitigating factors
+
+Review the security advisor for any mitigating factors that may allow you to continue using the package with the vulnerability. The vulnerability may only exist when the code is used on a specific framework, operating system, or a special function is called.
```

Comment:
Should we have a note that tells customers that at this time there's no way for them to suppress a specific advisory, or ignore all advisories for a specific package?  It's clearly something we need to implement, but at this time we don't even have a design for it.

edit: alternativly, have a link to the `#excluding-advisories` heading

Reply:
This should be covered enough by that section for now. We will await docs issues or more feedback first.